benchmarks/torchbench_model.py

```diff
@@ -124,6 +124,22 @@
     "vision_maskrcnn": [{}],
 }
 
+# This strict deny list denies tests that hold for too long and timeoout.
+STRICT_DENY_LIST = {
+    **{
+        "cm3leon_generate": [{
+            "test": "train",
+        }],  # no install.py
+        "hf_T5_generate": [{
+            "test": "train",
+        }],  # no install.py
```

Comment:
- I think these `STRICT_DENY_LIST` items (`{ "test": "train" }`) can be made into `DENY_LIST`, since these models fail to run training even on inductor 
- Shouldn't these be checking for `{ "xla": "PJRT" }`, since those are the ones that timeout (inductor doesn't)?

Reply:
I just tried the eager mode for GPU with `xla:null,dynamo:null` also, it passes for those two models. Did you see the failing somewhere?

Reply:
In addition, LTC mode with `xla:PJRT,dynamo:null` with cause timeout.

Reply:
- `xla:PJRT,dynamo:null`: both timeout
- `xla:PJRT,dynamo:openxla`: both end up with the same issue https://github.com/pytorch/xla/issues/5967

Reply:
That's true, let's just skip PJRT backend as you mentioned.

Reply:
Moved `cm3leon_generate` and `hf_T5_generate` from strict deny list to general list.